src/components/UserList/CardScroll.js

```diff
@@ -80,16 +80,9 @@ export default class CardScroll extends Component {
   controlScroll(e) {
     const event = e.nativeEvent;
     const interval = Dimensions.get('window').width - 57;
-
-    let offsetX = Math.ceil(event.contentOffset.x.toFixed(0));
-    let leftover = offsetX % interval;
-    if (leftover > 0 && leftover < 100) {
-      // Swipe right(iOS doesn't work)
-      // this.resetListView.scrollTo({x: offsetX - leftover + interval});
-    } else if (leftover > (interval - 100)) {
-      // Swipe left(iOS doesn't work)
-      // this.resetListView.scrollTo({x: offsetX - leftover});
-    }
+    let index =  Math.round(event.contentOffset.x / interval);
+    if (index > this.props.dataSource._cachedRowCount - 1 ) index -= 1;
+    this.resetListView.scrollTo({x: (Math.round(event.contentOffset.x / interval)) * interval, y: 0, animated: true});
```

Comment:
@heartjigi  Please!
this.resetListView.scrollTo({
  x: (Math.round(event.contentOffset.x / interval)) * interval,
  y: 0,
  animated: true,
});